install/configuring_inventory_file.adoc

```diff
@@ -1104,6 +1104,25 @@ openshift_hosted_routers=[{'name': 'router1', 'certificate': {'certfile': '/path
 
 include::install/topics/glusterfs_intro.adoc[]
 
+You configure {gluster} clusters using variables, which interact with the {product-title} clusters.
+The variables reside in the `[OSEv3:vars]` group and include host variables, role
+variables, and image name and version tag variables.
+
+The `glusterfs_devices` variable is a host variable that defines the list of block devices
+that can be managed as part of the {gluster} cluster. Each host in your configuration must
```

Comment:
s/that can be managed/to manage

I'd rewrite this as active you-centered statements.

Reply:
Rewritten. Thanks.